css/style.css

```diff
@@ -17,3 +16,171 @@ body {
  */
 
 
+/* default styling */
+* {
+   margin: 0;
+   padding: 0;
+   box-sizing: border-box;
+}
+
+li {
+   list-style: none;
+   font-size: .9em;
+}
+
+hr {
+   border: none;
+   background: #ddd;
+   height: .5px;
+}
+
+/* header */
+/* nav bar */
+.header {
+   height: 100vh;
+}
+
+.nav-bar {
+   height: 8vh;
+   background-color: #fff;
+   display: flex;
+   justify-content: space-between;
+   align-items: center;
+   padding: 0 5em;
+}
+
+.nav-bar img {
+   width: 15px;
+}
+
+li:first-child>a {
+   font-weight: 600;
+}
+
+.nav-bar a {
+   text-decoration: none;
+   font-weight: 400;
+   color: #444;
+}
+
+.nav-bar a:hover {
+   color: red;
+}
+
+.nav-list {
+   display: flex;
+   justify-content: space-around;
+   align-items: center;
+   gap: 2em;
+}
+
+/* intro section */
+.intro {
+   background: url(../img/first-background.jpg);
+   background-size: cover;
+   display: flex;
+   justify-content: center;
+   align-items: center;
+   flex-direction: column;
+   height: 92vh;
+   color: #fff;
+}
+
+.intro h1 {
+   font-weight: 400;
+}
+
+.intro h2 {
+   font-weight: 200;
+}
+
+.learn-btn {
+   margin-top: 2.5em;
+   padding: .45em;
+   background: rgb(216, 142, 5);
```

Comment:
This button should be orange (rgb(224,99,58)), yours is yellow. 
Suggestion: There's a Chrome extension, Eye Dropper, it can help to pick up colours off of webpages.

Reply:
Thanks, I will take a look on that.

Reply:
Thanks, I will take a look on that.